shuup/admin/modules/suppliers/views/list.py

```diff
@@ -33,6 +35,9 @@ class SupplierListView(PicotableListView):
         Column("module_identifier", _(u"Module"), display="get_module_display", sortable=True)
     ]
 
+    def get_queryset(self):
```

Comment:
I guess you could return all suppliers when superuser, like you did on `edit.py`

Reply:
This is already the same logic we use with other admin modules. Also I guess it makes it easier for superuser since the filter for shop in picotable is not the easiest task.